tests/Template/ClassTemplateTest.php

```diff
@@ -2038,11 +2038,6 @@ class A {
                         public function __construct(Foo $v) {
                             $this->vector = $v;
                         }
-
```

Comment:
Why are you removing this? The test isn't testing anything proper without this yield

Reply:
Foo is not an iterable, and cannot be yielded from.
In fact, if we try to iterate over the Generator returned by this method, "Can use "yield from" only with arrays and Traversables" will be returned.

Reply:
Oh ok, then would you mind making `Foo` something that can be yielded from? Or skip the test entirely my renaming it to `SKIPPED-<testname>`

Reply:
Done!